samza-core/src/main/java/org/apache/samza/config/JobConfig.java

```diff
@@ -132,7 +132,7 @@
 
   // Auto-sizing related configs that take precedence over respective sizing confings job.container.count, etc,
   // *only* when job.autosizing.enabled is true. Otherwise current behavior is maintained.
-  private static final String JOB_AUTOSIZING_CONFIG_PREFIX = "job.autosizing."; // used to determine if a config is related to autosizing
+  public static final String JOB_AUTOSIZING_CONFIG_PREFIX = "job.autosizing."; // used to determine if a config is related to autosizing
```

Comment:
Could we use the 
public boolean isAutosizingConfig(String configParam)
method in JobConfig for this?

Reply:
Updated